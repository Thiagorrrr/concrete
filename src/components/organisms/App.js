import React, { Component } from 'react';
import Search from '../molecules/search';
import Results from '../molecules/results';
import { BrowserRouter } from 'react-router-dom'
import { Route, Switch, Redirect } from 'react-router';
class App extends Component {

  render() {

    return (
      <div className="content">
        <BrowserRouter>
          <Switch>
            <Route exact path="/" component={Search} />
            <Route exact path="/results" component={Results} />
          </Switch>
        </BrowserRouter>
      </div>
    )
  }
}
export default App;