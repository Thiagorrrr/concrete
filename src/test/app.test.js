import React from 'react';
import { render } from '@testing-library/react';
import App from '../components/organisms/App';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<App />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});