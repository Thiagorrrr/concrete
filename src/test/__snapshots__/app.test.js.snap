// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="content"
>
  <div
    className="search"
  >
    <h1
      className="search__title Text-Style-3 "
    >
      Github 
      <span
        className="search__title--highligth Text-Style "
      >
        search
      </span>
       
    </h1>
    <div
      className="form"
    >
      <form
        className="form__form"
        onSubmit={[Function]}
      >
        <input
          className="form__input shadow Text-Style-2 "
          onChange={[Function]}
        />
        <div
          className="form__btn-content"
        >
          <label>
            <svg
              fill="none"
              height="38"
              viewBox="0 0 37 38"
              width="37"
              xmlns="http://www.w3.org/2000/svg"
            >
               
              <path
                clipRule="evenodd"
                d="M30 10.816c0 5.974-4.843 10.817-10.817 10.817-2.574 0-4.937-.899-6.794-2.4l-10.86 10.86L0 28.562l10.85-10.85a10.772 10.772 0 0 1-2.483-6.897C8.367 4.843 13.209 0 19.183 0c5.974 0 10.816 4.843 10.816 10.816zm-2.164 0a8.653 8.653 0 1 1-17.306 0 8.653 8.653 0 0 1 17.306 0z"
                fill="#fff"
                fillRule="evenodd"
                filter="url(#a)"
                transform="translate(3.895 3.895) scale(.9737)"
              />
               
              <defs>
                 
                <filter
                  filterUnits="userSpaceOnUse"
                  height="37.09"
                  id="a"
                  width="37"
                  x="0"
                  y="0"
                >
                   
                  <feFlood
                    floodOpacity="0"
                    result="BackgroundImageFix"
                  />
                   
                  <feColorMatrix
                    in="SourceAlpha"
                    values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 255 0"
                  />
                   
                  <feOffset />
                   
                  <feGaussianBlur
                    stdDeviation="2"
                  />
                   
                  <feColorMatrix
                    values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.4 0"
                  />
                   
                  <feBlend
                    in2="BackgroundImageFix"
                    result="effect1_dropShadow"
                  />
                   
                  <feBlend
                    in="SourceGraphic"
                    in2="effect1_dropShadow"
                    result="shape"
                  />
                   
                </filter>
                 
              </defs>
               
            </svg>
            <input
              className="form__btn"
              type="submit"
              value=""
            />
          </label>
        </div>
      </form>
    </div>
  </div>
</div>
`;
